spec/support/coronavirus_feature_steps.rb

```diff
@@ -305,7 +310,7 @@ def when_i_visit_the_new_timeline_entry_page
   visit "/coronavirus/landing/timeline_entries/new"
 end
 
-def then_i_see_the_create_timeline_entry_form
+def then_i_see_the_timeline_entry_form
```

Comment:
I liked it having `create` in better!

Reply:
I started to add another method called `then_i_see_the_edit_timeline_entry_form` until I realised they do the same thing.